test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -643,7 +643,7 @@ TEST_F(SingleBalancerTest, Fallback) {
   // Wait until the serverlist is ready.
   gpr_sleep_until(second_call_time);
```

Comment:
Instead of sleeping here, I suggest sending RPCs until you see that one of the backends returned by the balancer has gotten its first RPC:

```
do {
  SendRpc(kMessage_, 1);
} while (backend_servers_[kNumBackendInResolution].service_->request_count() == 0);
```

Once that loop finishes, you know that the serverlist has been processed, at which point you can call `ResetCounters()` on the backends and then do another loop showing that the backends returned by the balancer are getting traffic but the backends returned by the resolver are not.

Reply:
Accepted.

(I can't find this comment in the files during my review, maybe because it's outdated, so I replied directly.)